### PR TITLE
Added tests to methods in consolegui package. Added refactoring TODOs that need resolving.

### PR DESCRIPTION
**Pull request by paulslss300**

### CODE REVIEW

**Comment by Winnie1029** [comment]
The tests look great!

**Comment by Enidx** [comment]
The tests looks good, approve for merge

File: src/main/java/consoleforgui/HelperConsole.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * by ExistingUserController and NewUserController
  */
 public class HelperConsole {
-
+// TODO: several methods are never used (or the methods calling them are never used), refactor if time allows.
```
Comment (Jennylin331431) [inline src/main/java/consoleforgui/HelperConsole.java:15]: Yeah some of the methods were only used for in line commands, will definitely need to delete after GUI fully finished!

File: src/test/java/TestGuiHelperConsole.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import consoleforgui.HelperConsole;
+import org.junit.Test;
+
+
+public class TestGuiHelperConsole {
```
Comment (Jennylin331431) [inline src/test/java/TestGuiHelperConsole.java:5]: looks good to me, covers lots of cases !

**Review by Jennylin331431 — COMMENTED** [review]
Looks good to me!

**Pull request merged**